### PR TITLE
Feature#60/add partial view component for sold products

### PR DESCRIPTION
**Pull request by EliasUR**
# Description
What did you implemented/Why did you implemented this:
 - I´ve added a styled sold products view to show users their sales history
  
  ## Screenshots 
image

Closes: #60

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -23,7 +23,8 @@ def edit
   def create
     @post = Post.new(post_params)
     @post.type_id = 1
-
+    @post.user_id = current_user.id
+    puts @post.user_id
```
Comment (germanruzca) [inline app/controllers/posts_controller.rb:27]: Do you need a `puts` here?

**Review by germanruzca — APPROVED** [review]
Fix the conflicts

**Review by AlejandroCuriel — APPROVED** [review]
LGTM :rocket:

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (EliasUR) [reply app/controllers/posts_controller.rb]: I forgot that, thanks I´ve alredy delete it.

**Review by germanruzca — APPROVED** [review]

**Review by Javi1499 — APPROVED** [review]
LGTM

**Pull request merged**